### ISSUE TITLE
some planter images are broken

### ISSUE THREAD

**Issue by BenTheElder** [kubernetes/test-infra#7779]
/area planter
/kind bug

reported by @ixdy:
```
$ ../test-infra/planter/planter.sh
0.11.0: Pulling from k8s-testimages/planter
Digest: sha256:483752e01c7c98c30be5b1d82306b8b8037e0f68d1f7a5964f1969b3fd4c3d4e
Status: Image is up to date for gcr.io/k8s-testimages/planter:0.11.0
/bin/sh: 0: Can't open /entrypoint.sh
```
/assign

**Comment by BenTheElder** [comment]
I've pushed corrected images for `0.11.0`, `0.12.0`, will do the 0.13 rcs and then come back to 0.11 and 0.12 rcs

**Comment by BenTheElder** [comment]
pushed new images for `0.11.0`, `0.11.0rc1`, `0.11.0rc2`, `0.11.0rc3`, `0.11.0rc4`, `0.12.0`, `0.12.0rc1`, `0.12.0rc2`, `0.12.0rc3`, `0.13.0rc1`
